### PR TITLE
dashboard: add `Storage full ETA` panel

### PR DESCRIPTION
**Pull request by hagen1778**
The new panel suppose to help to estimate the time needed to run out of free
disk space.
Thx to @belm0 @hekmon

### CODE REVIEW

File: dashboards/victoriametrics.json [hunk 1]
```diff
@@ -1264,63 +1264,53 @@
...
+            "y": 3
           },
           "hiddenSeries": false,
-          "id": 34,
+          "id": 73,
           "legend": {
-            "avg": false,
-            "current": false,
+            "alignAsTable": true,
+            "avg": true,
```
Comment (belm0) [inline dashboards/victoriametrics.json:1273]: Avg of display range probably isn't useful, given that the query itself is over 1d range.

I'd suggest disabling the legend altogether.

**Review by belm0 — APPROVED** [review]
thank you!

would you attach a screenshot?

File: dashboards/victoriametrics.json (hunk 1 above)
Comment (hagen1778) [reply dashboards/victoriametrics.json:1273]: Good point!

**Comment by hagen1778** [comment]
> would you attach a screenshot?

Sure thing!
image

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
Thanks!

**Pull request merged**